src/evidence.cpp

```diff
@@ -469,14 +462,14 @@ void Courtroom::on_evidence_double_clicked(int p_id)
   ui_evidence_description->clear();
   ui_evidence_description->appendPlainText(f_evi.description);
   ui_evidence_description->setReadOnly(true);
-  ui_evidence_description->setToolTip(tr("Double-click to edit..."));
+  ui_evidence_description->setToolTip(tr("Click the pencil to edit..."));
```

Comment:
maybe I am being too meticulous, but if the evidence_edit widget wasn't loaded in the theme, the old tooltip should be used

Reply:
the old tooltip text will be inaccurate because I removed the functionality to double click to edit

Reply:
oh, _now_ it shows that you deleted the double click handler. ok